Frontend/src/components/Input/DateInput.svelte

```diff
@@ -28,29 +29,32 @@
     ["November", "Nov"],
     ["Dezember", "Dez"],
   ];
+  const TIMEZONE_OFFSET_MS = new Date().getTimezoneOffset() * 60000;
 
   function inTwoMonths() {
     const date = new Date();
     date.setMonth(date.getMonth() + 2);
     return date;
   }
 
-  function clear() {
-    isNone = true;
-    dateChosen = false;
-    selected = new Date();
+  export let calculate_from;
+  export let today = false;
+  export let quickset = {};
+  export let timeMillis = 0;
+  function setToday() {
+    let date = new Date();
+    timeMillis = date.getTime();
   }
 
-  export let timeMillis;
-  const TIMEZONE_OFFSET_MS = new Date().getTimezoneOffset() * 60000;
-
-  
-  let dateChosen = false;
-  let isNone = !timeMillis || timeMillis === 0;
-  let selected = isNone ? new Date() : new Date(timeMillis);
-
-  $: dateChosen, (isNone = !dateChosen && isNone);
-  $: timeMillis = isNone ? 0 : selected.getTime() - TIMEZONE_OFFSET_MS;
+  function addDays(days) {
+    if (calculate_from) {
+      timeMillis = calculate_from + days * 60 * 60 * 24 * 1000;
```

Comment:
But then here we could just use the current time (new Date().getTime()) instead of calculate_fom. Or is there any case where calculate from would not equal the current time?

Reply:
Yes, for now we can just use today.. And in congruency with that we can even get rid of the `today` prop, and just use `{0:"Heute"}` 😛 

I'm currently merging the changes from master in it and then it should be ready :) if at some point calculate_from will be useful we can still think about including it. Currently we don't really need it indeed.

Reply:
Ok great, yes we can always add more props later if necessary :)